draft-ietf-rats-eat.md

```diff
@@ -1221,8 +1221,8 @@ consumer.  Examples include key material or device management trees.  The consum
 may require an EAT to assess entity security state of the entity prior to provisioning.
 
 4 -- Certificate Issuance:
-: Certification Authorities (CA's) may require attestations prior to
-the issuance of certificates related to keypairs hosted at the entity.  An
+: Certification Authorities (CA's) may require Evidence and Attestation Results to make 
```

Comment:
```suggestion
: Certification Authorities (CA's) may require evidence and attestation results to make 
```

Reply:
I disagree with the use of Evidence here.   If it is intended to be correct, then something should explain why since this is against WG consensus on the arch doc.  It should just say "Attestation Results" in my view.

Reply:
I would expect the CA to receive evidence as a general rule. Re-reading that sentence, it should probably say "evidence or attestation results" but I'd not change it to attestation results alone. See https://datatracker.ietf.org/doc/html/draft-bweeks-acme-device-attest-01 and https://datatracker.ietf.org/doc/html/draft-ietf-lamps-key-attestation-ext-00 for specs that will convey evidence to a CA.

Reply:
For example, in Veracruz, the CA receives evidence (embedded in a CSR, IIRC) extracts it and forwards it to the "background check" verifier.  The same model is used in [ACME's key attestation](https://datatracker.ietf.org/doc/draft-bweeks-acme-device-attest/01/), where the webauthn "attestation object" ends up in the verification challenge reply.

Reply:
Thomas's point is that the CA may "receive" evidence as opaque and forward it on to a verifier in a background check scenario.  It then "receives" as non-opaque, and processes, the attestation results.  I was interpreting receives as the latter sense, whereas I think Carl might be using it in the former sense.   However, the text in the PR does not use the word "receives", it uses the word "requires", which to me is stronger in the former sense.   But it would be best to be explicit if you're referring to the Background Check model, such as

```suggestion
: Certification Authorities (CA's) may require Attestation Results (which in a Background Check model might require receiving Evidence to be passed to a Verifier) to make 
```

Reply:
I don't really think we want to mention attestation evidence and the background check model every time an RP wants/requires/gets/needs attestation results.